templates/studentAffectedByModule.html

```diff
@@ -0,0 +1,16 @@
+$def with (moduleCode, aysem, student_lst)
```

Comment:
Cannot visualize this page well due to lack of data showing up.

Also, please add a page description to this page, seeing as the word **affected** is considerably vague.